File: abc/abc143/a.cpp
// https://atcoder.jp/contests/abc143/submissions/26225639

#include <bits/stdc++.h>
using namespace std;
int main() {
  int a, b;
  cin >> a >> b;
  if(a <= 2 * b) cout << 0;
  else cout << a - 2 * b;
}

File: abc/abc143/a.java
// https://atcoder.jp/contests/abc143/submissions/43246168

import java.util.Scanner;

class Main {
  public static void main(String args[]) {
    Scanner stdIn = new Scanner(System.in);

    int a = stdIn.nextInt();
    int b = stdIn.nextInt();

    if(a <= b * 2) System.out.println(0);
    else System.out.println(a - b * 2);
  }
}
